ui/Screen/Onboarding/Success.svelte

```diff
@@ -45,8 +45,9 @@
     <p
       style="text-align: center; width: 23.13rem; margin: 1.75rem 0 1.75rem 0;
       color: var(--color-foreground-level-6);">
-      This is your handle — it's unique to you!<br /> Click to copy it and share
-      with others so that they can find you.
+      This is a <span class="typo-text-bold">peer address</span>! It's unique to
```

Comment:
Ah ok, seems we're going with "address" cool cool cool

Reply:
It's either going to be peerID or peer address pending my convo with @cloudhead today. Just was trying out peer address in the copy to see how it feels

Reply:
I'd personally advocate for "PeerId" since that's how it will show up in radicle-link documentation. And the fact that an address in the network will be more like a URL which will be a combination of PeerId and Urn.

Reply:
It's "Device ID" [now](https://github.com/radicle-dev/radicle-upstream/pull/1206/files#diff-35f363d0d1e254ca6f35454c30107f5fffc88b5da9be72a77e892d1de8e27202R48-R50) as per https://github.com/radicle-dev/radicle-upstream/pull/1206#issuecomment-724317000.